### PR TITLE
feat-navigation

### PR DESCRIPTION
**Pull request by abhij1607**
### Desktop view
image
### Mobile view
image

### CODE REVIEW

File: components/nav/nav.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+.top-nav .navigation {
+  flex-grow: 1;
+  align-items: center;
+  display: flex;
+}
+.navigation .nav-brand {
+  font-weight: 500;
+  font-size: 2rem;
```
Comment (stutisk) [inline components/nav/nav.css:19]: you can assign variables for font-size and font weight

**Review by stutisk — COMMENTED** [review]
else code looks good to me , great work abhi!

**Review by shuklahimanshu81 — COMMENTED** [review]
Good work! can be merged.

File: components/nav/nav.css (hunk 1 above)
Comment (abhij1607) [reply components/nav/nav.css:19]: thanks for the suggestion, I'll try to apply this later on

**Pull request merged**